doc/reference_guide/source/conf.py

```diff
@@ -21,13 +21,13 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Reference Guide'
-copyright = '2019, R. W. Ford, J. Henrichs, I. Kavcic, A. R. Porter, S. Siso'
+copyright = '2019-2020, R. W. Ford, J. Henrichs, I. Kavcic, A. R. Porter, S. Siso'
 author = 'R. W. Ford, J. Henrichs, I. Kavcic, A. R. Porter, S. Siso'
 
 # The short X.Y version
 version = ''
 # The full version, including alpha/beta/rc tags
-release = '1.7.0'
+release = '1.9.1'
```

Comment:
The wiki talks about updating doxygen.config on a release. Would that update this file, or does this need to be done manually as well? If so, it should be added to the wiki. And something to say to check the copyright year too.

Reply:
Oh yes, done.